src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems

```diff
@@ -9,23 +9,11 @@
     <Import_RootNamespace>AnalyzerDriver</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)AnalyzerDriverHelper.cs">
```

Comment:
I did an exhaustive search for StyleCop in the GitHub code and it came up clean after this change.  Going to do a corresponding one in TFS once this is submitted.